src/02_datatypes.py

```diff
@@ -9,13 +9,14 @@
 """
 
 x = 5
-y = "7"
+y: str = "7"
 
 # Write a print statement that combines x + y into the integer value 12
 
 # YOUR CODE HERE
-
+print(x + int(y))
 
 # Write a print statement that combines x + y into the string value 57
 
-# YOUR CODE HERE
\ No newline at end of file
+# YOUR CODE HERE
+print(str(5) + str(y))
```

Comment:
Another way to do this is:
`x = 5`
`y = "7"`
`print(x + int(y))`
`print(str(x) + y)`